scss/modules/matrix.scss

```diff
@@ -24,6 +24,17 @@
             margin: 0;
             border: none;
             white-space: nowrap;
+
+            > span:last-of-type{
+                display: none;
+            }
+        }
+
+        &.value:hover {
+            > span:last-of-type{
+                display: inline;
+                padding-left: 3px;
```

Comment:
Would move that after the `display:none` line. Then there is less happening on hover, which might be a little bit more performant.